apps/wallet-mobile/src/Dashboard/WithdrawStakingRewards/TransferSummary/TransferSummary.tsx

```diff
@@ -51,8 +51,8 @@ export const TransferSummary = ({wallet, unsignedTx}: {wallet: YoroiWallet; unsi
 const withdrawalInfo = (unsignedTx: YoroiUnsignedTx) => {
   const deregistrations = unsignedTx.staking?.deregistrations
   const withdrawals = unsignedTx.staking?.withdrawals
-  const withdrawalAmounts = Entries.toAmounts(withdrawals || {})
-  const deregistrationAmounts = Entries.toAmounts(deregistrations || {})
+  const withdrawalAmounts = Entries.toAmounts(withdrawals || [])
+  const deregistrationAmounts = Entries.toAmounts(deregistrations || [])
```

Comment:
prefer `??` for defaulting
```suggestion
  const withdrawalAmounts = Entries.toAmounts(withdrawals ?? [])
  const deregistrationAmounts = Entries.toAmounts(deregistrations ?? [])
```